sigstore-public-deployment-spec.md

```diff
@@ -164,12 +164,9 @@ Transparency logs are subject to sharding, usually once per year to keep the log
   * Root signing events are expected to occur about every 4-5 months.
 * Targets
   * Secured by the same HSM keys as the root to minimize the number of offline keysets.
-* Snapshot
-  * A GCP KMS snapshotting key.
-  * The snapshot ensures consistency of the metadata files. It has a lifetime of 3 weeks and is re-signed by a GitHub workflow.
-* Timestamp
-  * A GCP KMS timestamping key.
-  * The timestamp indicates the freshness of the metadata files. It has a lifetime of 1 week and is re-signed by two GitHub workflows.
+* Snapshot & timestamp
+  * Snapshot and timestamp roles ensures consistency and freshness of the artifacts
+  * A new timestamp is signed daily by a GCP KMS key: a timestamp older than a week will not be accepted by clients
 
 See [sigstore/root-signing repository](https://github.com/sigstore/root-signing) for additional implementation details.
 
```

Comment:
We might want to point out the TUF client spec doc below is outdated, particularly with the custom metadata.

Reply:
yeah... the client spec really should be replaced but one thing at a time. I added a note